roles/edpm_nftables/tasks/service-bootstrap.yml

```diff
@@ -32,7 +33,8 @@
         - iptables.service
         - ip6tables.service
       when:
-        - ansible_facts.services["{{ item }}"] is defined
+        - ansible_facts.services[item] is defined
+        - ansible_facts["services"][item]["status"] != "not-found"
```

Comment:
ansible_facts.services[item]["status"] != "not-found" for consistency

Reply:
daaaaamn, thanks @karelyatin